invoicer_project/static/js/login_request.js

```diff
@@ -0,0 +1,124 @@
+const host = "http://127.0.0.1:8000";
+const emailField = document.getElementById("email_input_lg_pg");
+const passwordField = document.getElementById("password_input_lg_pg");
+
+const fieldList = [emailField, passwordField];
```

Comment:
Don't understand the idea...(
Could you specify?

Reply:
fieldList occurs only in one place, did this instead of using fieldList:
`function removeAllErrorAttributes() {
    for (let item of [emailField, passwordField]) {
        item.removeAttribute("error");
        item.removeAttribute("errorText");
    }
}`